preupg/settings.py

```diff
@@ -43,6 +43,15 @@
 cache_dir = "/var/cache/preupgrade"
 log_dir = "/var/log/preupgrade"
 
+# file with meta info
+properties_ini = "properties.ini"
+# section inside ini file
+section_name = "preupgrade-assistant-modules"
+# key which specify source major OS version
+src_version_key ="srcMajorVersion"
+# key to specify destination major OS version
+dest_version_key ="destMajorVersion"
```

Comment:
Why warnings? The first part looks like if we're trying to negotiate the format with user.

We don't (and should not) provide any flexibility here. We have trivial file with prescribed mandatory format and placement. If user makes a typo, there's literally 2-3 obvious places to look at.

IMO all this needs is:

    try:
        parse properties.ini
        read [preupgrade-assistant-modules].srcMajorVersion
        read [preupgrade-assistant-modules].dstMajorVersion
    except AnyError:
        crash with message such as "missing mandatory meta-data key: srcMajorVersion"

Ie. make it easy for user to get the gist of what's wrong with the set (meta-data) and give them something to grep (srcMajorVersion).